features/admins/admin_works.feature

```diff
@@ -32,6 +32,27 @@ Feature: Admin Actions for Works, Comments, Series, Bookmarks
       | superadmin       |
       | legal            |
       | policy_and_abuse |
+  
+    Scenario Outline: Can hide works already marked as spam
```

Comment:
This scenario title should also mention something regarding the correct email (so not the spam one) getting sent.